docs/dg/dev/troubleshooting/troubleshooting-performance-issues/redis-session-lock-excpetions.md

```diff
@@ -67,3 +88,39 @@ Explore [architecture performance guidelines](/docs/dg/dev/guidelines/performanc
 
 ### Leverage APIs
 Headless scenarios aren't usually impacted by Session Locking challenges. Evaluate if you can adjust calls that would normally target Yves to target APIs instead.
+
+### Do not use locks
```

Comment:
Let's rename to "Not using locks". Otherwise sounds like recommendation.

Reply:
done